tests/handlers/external_test.py

```diff
@@ -189,18 +189,20 @@ async def test_route_retire(
 
 @pytest.mark.asyncio
 async def test_simultaneous_orders(
-    client: AsyncClient, dossier: Dossier, mock_kubernetes: MockKubernetesApi
+    client: AsyncClient, dossier2: Dossier, mock_kubernetes: MockKubernetesApi
 ) -> None:
-    r = await client.post("/moneypenny/commission", json=dossier.dict())
+    # We need a different dossier so that the cache doesn't just immediately
+    #  return success and defeat the whole purpose.
+    r = await client.post("/moneypenny/commission", json=dossier2.dict())
```

Comment:
You should instead clear the cache between tests rather than make the tests dependent on each other or force tests to be written differently because data is cached from previous tests. That probably means an additional method on the moneypenny dependency to clear the cache, and then calling it after each test.

Reply:
Done.